lib/child_process.js

```diff
@@ -690,3 +683,15 @@ function sanitizeKillSignal(killSignal) {
                                    killSignal);
   }
 }
+
+module.exports = {
+  ChildProcess,
+  fork,
+  _forkChild,
+  exec,
+  execFile,
+  spawn,
+  spawnSync,
+  execFileSync,
+  execSync
+};
```

Comment:
Optional nit: Can you put these in alphabetical order? When these kinds of lists get long, that can be a help when scanning the code.

Reply:
ok